SPECS/rubygem-yajl-ruby/rubygem-yajl-ruby.spec

```diff
@@ -1,16 +1,17 @@
 %global debug_package %{nil}
 %global gem_name yajl-ruby
+# Versions 1.4.2 and 1.4.3 are not available as tags
+%global commit e8de283a6d64f0902740fd09e858fc3d7d803161
```

Comment:
In such cases, for Ruby gems we can use the sources from rubygems.org. In this case: https://rubygems.org/downloads/yajl-ruby-1.4.3.gem. The gems contain the source, from which we can build it. See `rubygem-ffi.spec` or `facter.spec` for example.

Having said that, this projects seems abandoned and I wouldn't expect many updates here, so we may not gain much from keeping the updates simple by not having to update the commit hash. If you're going to stick to the hash, however, please add a short comment saying how you figured out that this is the right on, so that it's easy to update in the future, in case the project comes back to life.

Reply:
Will add a comment, but in this case I just looked at the version file https://github.com/brianmario/yajl-ruby/blob/master/lib/yajl/version.rb. The advisories say its fixed in 1.4.2, and actually fixed in 1.4.3, so I grabbed 1.4.3. The master branch seems to be mostly security fixes in the last while.

Reply:
I'll keep in mind the rugyegems.org path for future, but since the src tar is already up I'll leave this one as a github link.